frameworks/PHP/laravel/laravel-pripple.dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM php:8.3-cli
+
+RUN apt-get update -yqq >> /dev/null
+RUN apt-get install -y libevent-dev \
+    libssl-dev \
+    pkg-config \
+    build-essential \
+    unzip >> /dev/null
+
+RUN docker-php-ext-install pdo_mysql \
+    opcache \
+    posix \
+    pcntl \
+    sockets >> /dev/null
+
+RUN pecl install event >> /dev/null
+
+RUN docker-php-ext-enable pdo_mysql opcache posix pcntl sockets
+RUN docker-php-ext-enable --ini-name zz-event.ini event
+RUN echo "opcache.enable_cli=1" >> /usr/local/etc/php/conf.d/docker-php-ext-opcache.ini
+RUN echo "opcache.jit=1205" >> /usr/local/etc/php/conf.d/docker-php-ext-opcache.ini
+RUN echo "opcache.jit_buffer_size=128M" >> /usr/local/etc/php/conf.d/docker-php-ext-opcache.ini
+
+# Initialize
+WORKDIR /laravel
+COPY --from=composer --link /usr/bin/composer /usr/local/bin/composer
```

Comment:
Please, move the COPY composer  before or after the steps WORKDIR and COPY . .
Now it is in the middle.